src/autosub.py

```diff
@@ -773,7 +773,7 @@ def check_init_ressources(config_file):
     config = configparser.ConfigParser()
 
     try:
-        config.read_file(open(opts.configfile))
+        config.readfp(open(opts.configfile))
```

Comment:
readfp has been deprecated since Python 3.2 and was removed in Python 3.12 (October 2023). The function that shall be used is read_file